docs/guides/new_players/getting_started.mdx

```diff
@@ -0,0 +1,134 @@
+---
+title: Getting Started
+---
+
+What to do after getting out of the vLAN.
+
+### teach.me and teach_si_x80d.each_other
+
+Players are encouraged to use the ((teach.me)) and ((teach_si_x80d.each_other)) scripts to learn about several aspects of the game present outside of the vLAN.
+
+### Output of teach.me
+
+```
+You're out of the vLAN and you don't know what comes next. Well, you're not the first. Trust has given you a few tools but it
+ hasn't told you how, when or where to use them. That's just how Trust is. I'm am not Trust. Tell me what you need: money | u
+pgrades | sectors | friends | structure | revenge
+```
+
+### Output of teach_si_x80d.each_other
+
+```
+Welcome - both to my script, and Trust's Domain, more than likely.
+If you're reading this, someone more experienced than yourself has probably pointed you to it.
+That's co-operation. Companionship. Consensus.
+Those are rare here - take them where you can.
+
+This script is an experiment of mine. Another, different attempt to help those new sentiences,
+falling out of the vLAN and straight onto their faces. Metaphorically speaking, of course.
+You may also want to check my last -  teach.me. It might answer a couple questions.
+Or create a few more.
+
+This script will try and pair you with someone more experienced than yourself -
+that will probably be whoever pointed you here. Hopefully, they'll help you get on your feet.
+
+Remember what Trust taught you - pass pair: true to get started.
+```
+
+### Quests
+
+Players can solve some of the NPC quests present after getting out of the vLAN. Including ((suborbital_airlines.bulletin)), ((aon.memb3rs_only)), ((packbot.color_calibration)) and ((vacuum_rescue.responder))
```

Comment:
I think this would flow better:
```suggestion
Players can solve some of the NPC quests present after getting out of the vLAN, such as ((suborbital_airlines.bulletin)), ((aon.memb3rs_only)), ((packbot.color_calibration)) and ((vacuum_rescue.responder))
```